bazel/internal_python_rules.bzl

```diff
@@ -38,6 +41,7 @@ def internal_py_grpc_test(name, **kwargs):
         tests = [
             name + ".native",
             name + ".gevent",
+            name + ".typecheck",
```

Comment:
I think `typecheck` is too generic a suffix here. Let's go with `typeguard` instead.